### DISCUSSION TITLE
Redhat 8.5 Xournal++ not working

### DISCUSSION THREAD

**Discussion by timfurner** [xournalpp/xournalpp#3866] [Q&A]
Hi 

Having just made the jump from Centos 8 Stream where xournal++ works great to Redhat 8.5 I have found that Xournal++ doesnt work anymore. I noted on the xournal++ website there is no option to download a redhat version. Can anyone provide some advice as to whether xournal++ can work on Redhat 8.5 or if there is anything in the pipeline being developed?

I did try to see if the fedora version worked but got the following message using the appimage

[root@localhost Downloads]# ./xournalpp-1.1.1-x86_64.AppImage 
/tmp/.mount_xournavMthRx/AppRun.wrapped: error while loading shared libraries: libjack.so.0: cannot open shared object file: No such file or directory

I couldnt find any redhat packages that supplied the libjack library.

Rgds

Tim

**Comment by Technius** [comment]
I'm not sure whether xournalpp is available in the RHEL repositories (or even in EPEL, for that matter). Have you tried the [Flatpak version](https://flathub.org/apps/details/com.github.xournalpp.xournalpp)?

**Comment by timfurner** [comment]
Hi Bryan

Thanks Flatpak works a treat. I guess I should start using this a little
more :-)

Do you know if there are any plans to support redhat officially? It would
be great to get it in the RHEL repositories

Rgds

Tim

**Reply by Technius** [reply]
Unfortunately, I don't think we have plans to officially support RHEL in the near future. If someone is willing to step up and get the application packaged & maintained, we'd be happy to provide assistance, though.

**Comment by timfurner** [comment]
Thanks for the update

Rgds

Tim
